liberate-formula/README.md

```diff
@@ -1,14 +1,14 @@
 # Liberate formula
 
-This formula converts your systems from EL clones, Like CentOS 7 or even RHEL 9, to SUSE Liberty Linux. This method is meant to perform the conversion during the minion onboarding on [Uyuni](https://www.uyuni-project.org/) / [SUSE Manager](https://www.suse.com/products/suse-manager/), which is a tool for system management, patching deployment and automation.
+This formula converts your systems from EL clones, like CentOS 7 or even RHEL 9, to SUSE Liberty Linux. This method is meant to perform the conversion during the minion onboarding on [Uyuni](https://www.uyuni-project.org/) / [SUSE Manager](https://www.suse.com/products/suse-manager/), which is a tool for system management, patching deployment and automation.
 
 This is the result of the Hackweek 23 project 
 [Use Uyuni to migrate EL linux to SLL](https://hackweek.opensuse.org/23/projects/use-uyuni-to-migrate-el-linux-to-sll).
 
 ## Quickstart
 
 ### Install the formula
-We will be installing the formula from an RPM created in the Open Build Service
+We will be installing the formula from an RPM created in the Open Build Service.
```

Comment:
Re-reading it now, I think we should change or remove this sentence. Now the `liberate-formula` is part of the usual SUSE Manager channels and it doesn't require adding an additional one, like this wording seems to imply. What do you think?

Reply:
Yes, thanks for the proposal!